plugins/module_utils/foreman_helper.py

```diff
@@ -394,7 +394,7 @@ def find_resource(self, resource, search, name=None, params=None, failsafe=False
         return result
 
     def find_resource_by_name(self, resource, name, **kwargs):
-        search = 'name="{0}"'.format(name)
+        search = 'name="{0}"'.format(to_bytes(name))
```

Comment:
having `bytes` here seems weird to me. shouldn't it be `native`?

Reply:
or rather, text, looking at https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/net_tools/basics/uri.py#L477

Reply:
Yes, you are right.
However, I currently have to evaluate if the change is needed altogether, because after running my tests it seems to work again without this change as well :man_shrugging:
Encoding problems are weird.